src/apps/MobileApp/MobileClient.js

```diff
@@ -1051,6 +1051,10 @@ class MobileClient extends Component {
     // Subscribes selectively to different stream types |subscribeToVideo|, |subscribeToAudio|, |subscribeToData|.
     // This is required to stop and then start only the videos to save bandwidth.
     makeSubscription = (newFeeds, feedsJustJoined, subscribeToVideo, subscribeToAudio, subscribeToData) => {
+      const {user: {role}} = this.state;
```

Comment:
move to call site. Should not be part of `makeSubscription`. There is an argument for that: `subscribeToAudio`

Reply:
Fixed.